content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -221,12 +221,12 @@ following Pod-specific DNS policies. These policies are specified in the
   for more details.
 - "`ClusterFirst`": Any DNS query that does not match the configured cluster
   domain suffix, such as "`www.kubernetes.io`", is forwarded to the upstream
-  nameserver inherited from the node. Cluster administrators may have extra
+  nameserver inherited from the DNS Service's node. Cluster administrators may have extra
```

Comment:
I would change this even more, like:

"""
...is forwarded to an upstream nameserver by the DNS server.
"""

Reply:
Done! Thanks for the improved wording.